File: app/src/main/java/com/example/testgithub/db/GithubDb.java
package com.example.testgithub.db;


import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;


import com.example.testgithub.model.OrgRepos;

/**
 * Main database description.
 */
@Database(entities = {OrgRepos.class}, version = 1)
public abstract class GithubDb extends RoomDatabase {

    abstract public OrgRepDao orgRepDao();

}

File: app/src/main/java/com/example/testgithub/db/GithubDb.kt
package com.example.testgithub.db


import com.example.testgithub.model.OrgRepos

import androidx.room.Database
import androidx.room.RoomDatabase

/**
 * Main database description.
 */
@Database(entities = [OrgRepos::class], version = 1, exportSchema = false)
abstract class GithubDb : RoomDatabase() {

    abstract fun orgRepDao(): OrgRepDao

}
